packages/footer/components/Navigation.tsx

```diff
@@ -52,21 +52,33 @@ export const NavigationSection = ({
    );
 };
 
-const NavigationColumn = ({ menu }: { menu: MenuType | null }) => {
+export const NavigationColumn = ({
+   menu,
+   ...otherProps
+}: {
+   menu: MenuType | null;
+}) => {
```

Comment:
We can import chakra's `TextProps` so we can state what we are expecting to be in `otherProps`
```suggestion
}: {
   menu: MenuType | null;
} & TextProps) => {
```